backend.native/tests/runtime/collections/hash_map0.kt

```diff
@@ -155,7 +155,6 @@ fun testEquals() {
     assertFalse(m == mapOf("a" to "1", "b" to "2", "c" to "5"))
     assertFalse(m == mapOf("a" to "1", "b" to "2"))
     assertEquals(m.keys, expected.keys)
-    assertEquals(m.values, expected.values)
```

Comment:
After changes in this commit `values` collection inherits `Object.equals()` and `Object.hashCode()` methods. 
Such behaviour is consistent with java `LinkedHashMap`

Reply:
Can be replaced with `m.values.toList()` and `expected.values.toList()`

Reply:
done